src/Service.Host/client/src/components/saHoldStories/SaHoldStory.js

```diff
@@ -131,21 +141,27 @@ function SaHoldStory({
                         ) : (
                             <Fragment />
                         )}
-                        <Grid item xs={6}>
-                            <InputField
-                                value={saHoldStory.reasonStarted}
-                                disabled={!creating()}
-                                onChange={handleFieldChange}
-                                propertyName="reasonStarted"
-                                rows={4}
-                                label="Reason Put on Hold"
-                                fullWidth
-                                error={!saHoldStory.reasonStarted}
-                                helperText={
-                                    !saHoldStory.reasonStarted ? 'You must provide a reason' : ''
-                                }
-                            />
-                        </Grid>
+                        {viewing() || creating() ? (
+                            <Grid item xs={6}>
+                                <InputField
+                                    value={saHoldStory.reasonStarted}
+                                    disabled={!creating()}
+                                    onChange={handleFieldChange}
+                                    propertyName="reasonStarted"
+                                    rows={4}
+                                    label="Reason Put on Hold"
+                                    fullWidth
+                                    error={!saHoldStory.reasonStarted}
+                                    helperText={
+                                        !saHoldStory.reasonStarted
+                                            ? 'You must provide a reason'
+                                            : ''
+                                    }
+                                />
+                            </Grid>
+                        ) : (
```

Comment:
if you don't have an alternative thing to render you can use {condition && html}